src/NuGet.Clients/NuGet.PackageManagement.UI/Options/PackageSourcesOptionsControl.resx

```diff
@@ -435,7 +353,7 @@
     <value>2, 0, 2, 0</value>
   </data>
   <data name="BrowseButton.Size" type="System.Drawing.Size, System.Drawing">
-    <value>48, 34</value>
+    <value>34, 22</value>
```

Comment:
Thanks for raising the concern. Are you sure this is a bug and/or a concern? What was the issue that resulted from this?

From a quick read of https://stackoverflow.com/a/37671798, it sounds like this is how WinForms handles scaling. If it's a concern for the source code to change, then we would have to perfectly match the host machine's resolution/scaling. So far, I'm not convinced there's a reason to worry about that.

Reply:
Found the PR you're mentioning: https://github.com/NuGet/NuGet.Client/pull/4027

I'm curious how smaller buttons made the dropdown disappear, or if there was actually another sneaky bug there.
I disabled my DPI scaling earlier, and VS designer still resized several controls. My understanding is this will happen unless my machine matches exactly that of whoever wrote this control years ago.

... will keep working with it ...

Reply:
The PR that had issues (#4027) may have been due to the addition of Padding. Padding will definitely skew control layout, and I don't think scaling/resolution will affect that.
Additionally, there are still Size changes in the PR that was merged, despite disabling DPI scaling.

I set my machine's DPI scaling to 100% on all screens, and the designer generated the same Size changes as with DPI scaling at 200%.

My conclusion is that Size changes were not the issue; padding changes were. This PR doesn't have any padding changes, so I think it's safe.